addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1720,9 +1720,14 @@ body.editor_enable.editor_has_snippets {
                         color: $o-we-fg-lighter;
                     }
                 }
-                > .oe_snippet_remove {
+                // TODO In master: restore `.oe_snippet_remove`.
```

Comment:
I am not sure to understand why we need this either:
- The field is required and can't be deleted: we show the icon in gray -> disabled 
- The field can be deleted -> The icon is red -> enabled
What's wrong?

Reply:
I guess we keep it then (given https://github.com/odoo/odoo/pull/98292#issuecomment-1302201848)